onboarding/models.py

```diff
@@ -18,6 +18,9 @@ def __str__(self):
 
 
 class CompanyQuestionAndAnswer(models.Model):
+    """
+
+    """
```

Comment:
it would be worth adding something here :)

Reply:
True (y)